packages/transformer/src/IModelImporter.ts

```diff
@@ -597,9 +638,28 @@ export class IModelImporter implements Required<IModelImportOptions> {
       options: this.options,
       targetDbId: this.targetDb.iModelId || this.targetDb.nativeDb.getFilePath(),
       doNotUpdateElementIds: CompressedId64Set.compressSet(this.doNotUpdateElementIds),
+      duplicateCodeValueMap: this._duplicateCodeValueMap ? Object.fromEntries(this._duplicateCodeValueMap) : undefined,
       additionalState: this.getAdditionalStateJson(),
     };
   }
+
+  public resolveDuplicateCodeValues(): void {
```

Comment:
why public

Reply:
We need a way to resolve all conflicts after all imports are done. `IModelTransformer.finalizeTransformation` is the place I do this.
Since we removed the option to enable this handling I made this private and exposed a more "generic" method `finalize()`.

Reply:
this is now private